_sass/_post.scss

```diff
@@ -30,3 +30,7 @@
   }
 
 }
+
+p img {
```

Comment:
Nop, just 
```css
.post-content { img { width: 100% } }
```
because in this mode  no matter if they are inside a `<p>` or a `<li>` or whatever, all the image should adjust to the content width :smile: .